readme.md

```diff
@@ -27,8 +27,8 @@ The page should match the design Pixel Perfect: all the sizes, colors and distan
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_moyo-header/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_moyo-header/report/html_report/)
+- [DEMO LINK](https://Korochenko.github.io/layout_moyo-header/)
+- [TEST REPORT LINK](https://Korochenko.github.io/layout_moyo-header/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="970" alt="image" src="https://github.com/user-attachments/assets/7693d2e1-c8e9-4b32-8443-8d2c93b1a633" />